eslint.config.mjs

```diff
@@ -0,0 +1,35 @@
+import bpmnIoPlugin from 'eslint-plugin-bpmn-io';
+
+const files = {
+  expose: [
+    '**/*.js',
+    '*.js',
+  ],
+  build: [
+    '*.mjs',
+  ]
+};
+
+/** @type {import('eslint').Linter.Config[]} */
+export default [
+  ...bpmnIoPlugin.configs.browser.map(config => {
+    return {
+      ...config,
+      ignores: files.build,
+      files: files.expose,
```

Comment:
```suggestion
```

We don't need this. JS files are included per default. (cf. [bpmn-js](https://github.com/bpmn-io/bpmn-js/blob/develop/eslint.config.mjs))